core/lib/serverActions.ts

```diff
@@ -58,4 +58,4 @@ export function useServerAction<T extends unknown[], U>(action: (...args: T) =>
 }
 
 export const didSucceed = <T>(result: T): result is Exclude<T, ClientException> =>
-	!isClientException(result);
+	typeof result !== "object" || (result !== null && Reflect.get(result, "error") === undefined);
```

Comment:
what the hell is this man hahaha

why is `Reflect` necessary? if possible i'd like to avoid

Reply:
Ah it isn't necessary. I use reflect when I'm too lazy to type things sometimes. I accidentally left it in